pkg/processor/processor.go

```diff
@@ -76,17 +76,20 @@ func RemoveOldImages(t *core.CleanupTask, kubeClient kubernetes.KubernetesClient
 		glog.V(10).Infof("Max Images is %d", t.MaxImages)
 		unusedOldImages := aws.FilterOldUnusedImages(t.MaxImages, images, usedImages[repoName])
 
-		if len(unusedOldImages) == 0 {
+		unusedImages := utils.ApplyKeepFilters(unusedOldImages, t.KeepFilters)
+		glog.Infof("Number of images after blacklist filter: %d", len(unusedImages))
```

Comment:
This is the only place it is ever called a blacklist filter which is rather confusing.